nav2_safety_nodes/include/nav2_safety_nodes/nav2_safety_node.hpp

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2021
```

Comment:
Add your name to all of these copyright lines. It should be `Copyright (c) 2021 <your name>`